### ISSUE TITLE
bug:  react-native-gesture-handler module was not found

### ISSUE THREAD

**Issue by ARMATAV** [t3-oss/create-t3-turbo#462]
### Provide environment information

```
  System:
    OS: macOS 13.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 81.92 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 18.16.0 - ~/.nvm/versions/node/v18.16.0/bin/node
    Yarn: 1.22.19 - ~/.nvm/versions/node/v18.16.0/bin/yarn
    npm: 9.5.1 - ~/.nvm/versions/node/v18.16.0/bin/npm
    pnpm: 8.6.11 - ~/Library/pnpm/pnpm
```

### Describe the bug

Followed the instructions in the repo then I did `cd apps/expo && eas build --profile development --platform ios` and I get

```
ERROR  [react-native-gesture-handler] react-native-gesture-handler module was not found. Make sure you're running your app on the native platform and your code is linked properly (cd ios && pod install && cd ..).
```

### Link to reproduction

N/A, default repo was all that was used

### To reproduce

1. Clone the repo
2. Go through the setup steps
3. `cd apps/expo && eas build --profile development --platform ios`
4. Scan the QR code on your phone
5. Start the dev server
6. App on phone returns `ERROR  [react-native-gesture-handler] react-native-gesture-handler module was not found. Make sure you're running your app on the native platform and your code is linked properly (cd ios && pod install && cd ..).`

**Comment by ARMATAV** [comment]
Fixed with `"react-native-safe-area-context": "4.6.3"` and `cd apps/expo && npx expo add react-native-gesture-handler` - I think maybe it should be a dep for the repo since it blocks `eas build`.

**Comment by juliusmarminge** [comment]
Fixed in [f17266c](https://github.com/t3-oss/create-t3-turbo/commit/f17266c33c718a50dc25e3ce810259194a2791b4). Thanks for pointing this out!
